Public/Src/Engine/Dll/Engine.cs

```diff
@@ -273,19 +273,21 @@ private FileContentTable FileContentTable
             Contract.Requires(configuration != null);
             Contract.Requires(initialConfig != null);
 
-            if (configuration.Distribution.IsGrpcEnabled)
-            {
-                bool grpcHandlerInliningEnabled = GrpcSettings.HandlerInliningEnabled;
+            bool grpcHandlerInliningEnabled = GrpcSettings.HandlerInliningEnabled;
 
 #if NET_CORE
-                // Handler inlining causing deadlock on the mac platform.
-                grpcHandlerInliningEnabled = false;
+            // Handler inlining causing deadlock on the mac platform.

```

Comment:
>// Handler inlining causing deadlock on the mac platform. [](start = 12, length = 57)

This will apply to windows builds too now that we switched them over to NetCore. Can we just remove the directive for simplicity?

Reply:
We should go and vet the code base for `#if NET_CORE` now anyway, and use `PLATFORM_OSX` where macOS specific adjustments were done (and `PLATFORM_LINUX` later).

Reply:
switched to PLATFORM_OSX

---
In reply to: [315298936](https://github.com/microsoft/BuildXL/pull/758#discussion_r315298936) [](ancestors = 315298936)

Reply:
switched to PLATFORM_OSX

---
In reply to: [315296948](https://github.com/microsoft/BuildXL/pull/758#discussion_r315296948) [](ancestors = 315296948)